src/elements/teaser/teaser.visual.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { html, nothing } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualDiffStandardStates,
+} from '../core/testing/private.js';
+import { waitForImageReady } from '../core/testing.js';
+
+import './teaser.js';
+
+const imageUrl = import.meta.resolve('../core/testing/assets/placeholder-image.png');
+
+describe(`sbb-teaser`, () => {
+  const loremIpsum: string = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer enim elit, ultricies in tincidunt
+    quis, mattis eu quam. Nulla sit amet lorem fermentum, molestie nunc ut, hendrerit risus. Vestibulum rutrum elit et
+    lacus sollicitudin, quis malesuada lorem vehicula. Suspendisse at augue quis tellus vulputate tempor. Vivamus urna
+    velit, varius nec est ac, mollis efficitur lorem. Quisque non nisl eget massa interdum tempus. Praesent vel feugiat
+    metus. Donec pharetra odio at turpis bibendum, vel commodo dui vulputate. Aenean congue nec nisl vel bibendum.
+    Praesent sit amet lorem augue. Suspendisse ornare a justo sagittis fermentum.`;
+  const longChip: string =
+    'This is a chip which has a very long content and should receive ellipsis.';
+
+  const visualStates = {
+    hasChip: [false, true],
+    withLongContent: [false, true],
+  };
+
+  const screenCombinations = [
+    { viewport: 'micro' as const, alignments: ['below'] },
+    { viewport: 'medium' as const, alignments: ['after-centered', 'after', 'below'] },
+  ];
+
+  for (const screenCombination of screenCombinations) {
+    describeViewports({ viewports: [screenCombination.viewport] }, () => {
+      for (const alignment of screenCombination.alignments) {
+        for (const visualDiffStandardState of visualDiffStandardStates) {
+          it(
+            `alignment=${alignment} ${visualDiffStandardState.name}`,
+            visualDiffStandardState.with(async (setup) => {
+              await setup.withFixture(
+                html`
+                  <sbb-teaser title-content="This is a title" href="#" alignment=${alignment}>
+                    <img slot="image" src=${imageUrl} alt="400x300" />
+                    This is a paragraph
+                  </sbb-teaser>
+                `,
+                { maxWidth: '760px' },
```

Comment:
are those 760px really needed? In list test it could be that it causes a croped picture (see right side) https://lyne-components-visual-regression-diff-pr2899.app.sbb.ch/compare/sbb-teaser/list=true_below

Reply:
ul tag adds an extra padding in lists, removed

Reply:
This test https://lyne-components-visual-regression-diff-pr2899.app.sbb.ch/compare/sbb-teaser/alignment=below_longChip=true doesn't show the ellipsis now. maybe here a max-width: 100% needed? or any idea?

Reply:
In the lastest build should be fixed